guides/source/action_text_overview.md

```diff
@@ -81,7 +81,7 @@ class Message < ApplicationRecord
 end
 ```
 
-Note that you don't need to add a `content` field to your `messages` table.
+**Note:** you don't need to add a `content` field to your `messages` table.
```

Comment:
Instead of `field` saying `database column content` sounds more readable to me. Please ignore if that doesn't sound okay to you. :)